src/screen.rs

```diff
@@ -64,6 +65,12 @@ impl <'a> Screen <'a>{
         self.ansi.set_position(line, 0);
 
         match *text {
+            Text::Colored(ref matching) => {
+                let (start, middle, end) = matching.range.slice(&matching.original);
```

Comment:
When I read the function `slice` earlier in the Substring impl I liked it, but here this name gets confusing. Concepts of slicing a string into parts vs rust's idea of memory slices get conflated in my head.

I wonder if the `slice` function on `Substring` could be renamed.. I'm not coming up with anything better than just `parts` currently